app/controllers/api/v1/lead_controller.rb

```diff
@@ -1,2 +1,38 @@
 class Api::V1::LeadController < ApplicationController
+
+
+    def create
+        @lead = Lead.create(lead_params)
+       
+        if @lead.valid?
+
+          render json: {status: :ok, message: 'lead creado correctamente' }
+        else
+          render json: {message: "ocurrio un error porfavor intente de nuevo", status: :error}
+        end
+      end
+
+
+      
+
+    def leadsProject
+        @leadProject = Lead.where(proyect_id: params[:projectId])
+      
+
+        if @leadProject.empty?
+
+            render json: {message: "ocurrio un error porfavor intente de nuevo", status: :error}
+        else
+            render json: {data: @leadProject, projectCounts:@leadProject.count, status: :ok}
```

Comment:
Use mejor .size o .length en lugar de .count.

> if you already load all entries, say User.all, then you should use length to avoid another db query
> if you haven't anything loaded, use count to make a count query on your db
> if you don't want to bother with these considerations, use size which will adapt